src/main/resources/hudson/matrix/MatrixBuild/ajaxMatrix.jelly

```diff
@@ -32,11 +32,12 @@ THE SOFTWARE.
       <j:set var="b" value="${p.run}" />
       <j:choose>
         <j:when test="${b==null}">
-          <l:icon class="icon-nobuilt icon-md" tooltip="${%Not run}" alt="${%Not run}"/>
+          <l:icon src="symbol-status-nobuilt" class="icon-md" tooltip="${%Not run}" alt="${%Not run}"/>
+          ${%Not configured}
```

Comment:
should this be "Not run" to match `tooltip` and `alt`?

Reply:
```suggestion
          ${%Not run}
```